string face
---
float64[] pose
bool visible
